markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+          \item Date: Year-Month (usually the completion/version date)
+          \item Software: Stata
+          \item Description: empty
+          \item Methods: \textcolor{red}{This is methods used in the script}. Usually empty
+          \item Dependencies: Here enter all packages necessary:\footnote{It may be preferable to enter this in Software, libraries: \textcolor{red}{Error: Must be an array}. Does not accept more than one entry, and also not a string. Behavior of the editor his random here}
+            \begin{lstlisting}
+              coefplot, distinct, lincomeest, lprobust.ado
+            \end{lstlisting}
+          \item Instructions: Write: "See readme in reproducibility package"
+          \item Repository: ICPSR, Harvard Dataverse, Journal webpage etc. For ICPSR
+                the correct description is
+                \begin{lstlisting}
+                  AEA Data and Code Repository (ICPSR)
+                  Reproducible Research Repository (World Bank)
+                \end{lstlisting}
+          \item Notes: \textcolor{red}{Here who verified the package}. 
+            \begin{lstlisting}[breaklines]
+            Checked for reproducibility externally at the journal.
+            Checked for reproducibility at Development Impact Evaluation (DIME), World Bank
+            \end{lstlisting}
+          \item License:
+            \begin{itemize}
+              \item PP
+              \begin{itemize}
+                \item ICPSR:
+                  \begin{lstlisting}[breaklines]
+                  Creative Commons Attribution 4.0 International (CC BY 4.0) License
+                  https://creativecommons.org/licenses/by/4.0/
+                  \end{lstlisting}
+              \end{itemize}
+              \item WP: Modified BSD3
+            \end{itemize}
+      \end{enumerate}
+  \item Repository: Harvard, ICPSR- Type empty - URL, Journal webpage
+    \begin{lstlisting}[breaklines]
+        AEA Data and Code Repository
+        https://www.openicpsr.org/openicpsr/aea
+    \end{lstlisting}
+  \item Reproducibility:
+      \begin{enumerate}
+            \item Technology environment: empty
+            \item Technology Requirements: Standard, Cluster...; include runtime if available
+            \item Reproduction instructions: No details:
+                \begin{lstlisting}[breaklines]
+                 A readme file with detailed instructions is part of the (external) reproducibility package.
+                \end{lstlisting}
+                 Short details:
+                  \begin{lstlisting}
+                    A readme file and a single do file, AP_final_AEJ. do are provided. 
+                  \end{lstlisting}
+        \end{enumerate}
+\end{enumerate}
+
+## Access and Rights
+\begin{enumerate}
+    \item Disclaimer: \textcolor{red}{Perhaps not necessary}
```

Comment:
not necessary. discussing with Legal now what we need for terms of use of the site but we can remove this from the metadata.